### PR TITLE
7.0 - Setup project with Travis, Coverage, added README and .gitignore

### PR DESCRIPTION
**Pull request by laetitia-gangloff**

### CODE REVIEW

**Comment by pedrobaeza** [comment]
:+1:

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+install:
+ - git clone https://github.com/gurneyalex/maintainer-quality-tools.git $HOME/maintainer-quality-tools
+ - export PATH=$HOME/maintainer-quality-tools/travis:$PATH
+ - $HOME/maintainer-quality-tools/travis/travis_install_nightly 7.0
+ - pip install coveralls flake8
+ - pip install unidecode
+
+services:
+  - postgresql
```
Comment (yvaucher) [inline .travis.yml]: Please remove those 2 lines

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+script:
+    - travis_run_flake8
+    - travis_run_tests 7.0 openerp_test
```
Comment (yvaucher) [inline .travis.yml]: openerp_test to be removed

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,23 @@
+language: python
+python:
+  - "2.7"
+
+virtualenv:
+  system_site_packages: true
+
+install:
+ - git clone https://github.com/gurneyalex/maintainer-quality-tools.git $HOME/maintainer-quality-tools
```
Comment (yvaucher) [inline .travis.yml]: replace gurneyalex by OCA as @gurneyalex work is now under OCA umbrella

File: .travis.yml [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+ - export PATH=$HOME/maintainer-quality-tools/travis:$PATH
+ - $HOME/maintainer-quality-tools/travis/travis_install_nightly 7.0
```
Comment (yvaucher) [inline .travis.yml]: this line can simply be 
    - travis_install_nightly 7.0

File: README.md [hunk 5]
```diff
@@ -0,0 +1,6 @@
+[![Build Status](https://travis-ci.org/OCA/product-attribute.svg?branch=7.0)](https://travis-ci.org/OCA/product-attribute)
+[![Coverage Status](https://img.shields.io/coveralls/OCA/product-attribute.svg)](https://coveralls.io/r/OCA/product-attribute?branch=7.0)
```
Comment (yvaucher) [inline README.md]: please use png badge to allow you to specify the version
See https://raw.githubusercontent.com/OCA/connector/7.0/README.md

**Comment by bwrsandman** [comment]
LGTM, only seems to be failing due to PEP8 which can be done in another commit

**Comment by laetitia-gangloff** [comment]
I am not sure it is only pep8 problems.

2014-07-09 10:39:58,559 7773 ERROR openerp_test openerp.modules.loading: module partner_custom_attributes: an exception occurred in a test
Traceback (most recent call last):
  File "/usr/lib/pymodules/python2.7/openerp/modules/loading.py", line 83, in load_test
    _load_data(cr, module_name, idref, mode, 'test')
  File "/usr/lib/pymodules/python2.7/openerp/modules/loading.py", line 109, in _load_data
    fp = tools.file_open(pathname)
  File "/usr/lib/pymodules/python2.7/openerp/tools/misc.py", line 173, in file_open
    return _fileopen(name, mode=mode, basedir=rtp, pathinfo=pathinfo, basename=basename)
  File "/usr/lib/pymodules/python2.7/openerp/tools/misc.py", line 220, in _fileopen
    raise IOError('File not found: %s' % basename)
IOError: File not found: partner_custom_attributes/test/partner_attribute_test.yml

**Comment by yvaucher** [comment]
Config is ok

PEP8 and tests can be fixed in other PR

**Comment by yvaucher** [comment]
Thanks for the work

**Pull request merged**